Makefile

```diff
@@ -6,11 +6,21 @@ define eachmod
 	@find . -type f -name go.mod -print0 | xargs -I '{}' -n1 -0 bash -c 'dir="$$(dirname {})" && echo "$${dir}" && cd "$${dir}" && $(1)'
 endef
 
+all: rm-docker-container-deps build-sidecar-image install
+
+rm-docker-container-deps:
+	docker rm -f testground-sidecar || true
+	docker rm -f testground-redis || true
+	docker rm -f testground-goproxy || true
+
+install:
+	go install ./...
+
 pre-commit:
 	python -m pip install pre-commit --upgrade --user
 	pre-commit install --install-hooks
 
-docker-ipfs-testground:
+build-sidecar-image:
```

Comment:
This doesn't just build the sidecar image, it builds testground entirely. Let's call this `make docker`.

We also need to update `.travis.yml`